src/app/give-help/material-aid-form/material-aid-form.component.html

```diff
@@ -0,0 +1,37 @@
+<mat-card>
+  <app-offer-form (submitOffer)="handleSubmit($event)">
+    <div class="row">
+      <div class="col-md-6">
+        <label
+          ><b>{{ "TYPE_OF_SUPPORT" | translate }}:</b></label
+        >
+        <mat-form-field appearance="outline">
+          <mat-select
+            placeholder="{{ 'SELECT_PLACEHOLDER' | translate }}"
+            [(ngModel)]="data.category"
```

Comment:
Please use Reactive forms
with form builder: https://angular.io/guide/reactive-forms#using-the-formbuilder-service-to-generate-controls